ironfish/src/rpc/routes/chain/getBlocks.test.ts

```diff
@@ -0,0 +1,94 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Assert } from '../../../assert'
+import { useBlockWithTx, useMinerBlockFixture } from '../../../testUtilities'
+import { createRouteTest } from '../../../testUtilities/routeTest'
+import { RPC_ERROR_CODES } from '../../adapters'
+
+describe('Route chain/getBlocks', () => {
+  const routeTest = createRouteTest()
+
+  it('Processes sequence inputs', async () => {
+    // Create a 3 block chain
+    const { chain } = routeTest
+    await chain.open()
+
+    const blockA1 = await useMinerBlockFixture(chain)
+    await expect(chain).toAddBlock(blockA1)
+    const blockA2 = await useMinerBlockFixture(chain)
+    await expect(chain).toAddBlock(blockA2)
+
+    // Get hash of blocks
+    const hash0 = chain.genesis.hash.toString('hex')
+    const hash1 = blockA1.header.hash.toString('hex')
+    const hash2 = blockA2.header.hash.toString('hex')
+
+    // Find block matching request sequence range
+    let response = await routeTest.client.chain.getBlocks({ start: 1, end: 4 })
+    expect(response.content.blocks).toHaveLength(3)
+    const blocks = response.content.blocks
+    expect(blocks[0]).toMatchObject({
+      block: {
+        hash: hash0,
+        sequence: 1,
+      },
+    })
+    expect(blocks[0]).toMatchObject({
+      block: {
+        hash: hash1,
+        sequence: 2,
+      },
+    })
+    expect(blocks[0]).toMatchObject({
+      block: {
+        hash: hash2,
+        sequence: 3,
+      },
+    })
```

Comment:
This looks like it's checking blocks[0] multiple times. I'm guessing it should be:

```suggestion
    expect(blocks[0]).toMatchObject({
      block: {
        hash: hash0,
        sequence: 1,
      },
    })
    expect(blocks[1]).toMatchObject({
      block: {
        hash: hash1,
        sequence: 2,
      },
    })
    expect(blocks[2]).toMatchObject({
      block: {
        hash: hash2,
        sequence: 3,
      },
    })
```